components/brave_rewards/resources/ui/components/adsBox.tsx

```diff
@@ -108,8 +108,8 @@ class AdsBox extends React.Component<Props, State> {
     if (adsData) {
       adsEnabled = adsData.adsEnabled
       adsUIEnabled = adsData.adsUIEnabled
-      notificationsReceived = adsData.adsNotificationsReceived
-      estimatedEarnings = adsData.adsEstimatedEarnings.toFixed(2)
+      notificationsReceived = adsData.adsNotificationsReceived || 0
+      estimatedEarnings = (adsData.adsEstimatedEarnings || 0).toFixed(2)
```

Comment:
we need to set it to 0 in case `adsEstimatedEarnings` is undefined, where it will be with old profiles